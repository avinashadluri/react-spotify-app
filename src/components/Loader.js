import React from "react";
import * as Styles from "./Styles";

class Loader extends React.Component {
    render() {
      return (
        <Styles.Loader>
          <Styles.LoaderText>Loading...</Styles.LoaderText>
        </Styles.Loader>
      );
    }
}

export default Loader;