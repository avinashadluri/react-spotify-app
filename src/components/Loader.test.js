import React from 'react';
import Loader from './Loader';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Loader />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});