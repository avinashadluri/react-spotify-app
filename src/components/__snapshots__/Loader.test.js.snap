// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-Axmtr gFKpTb"
>
  <p
    className="sc-AxmLO bmJSYo"
  >
    Loading...
  </p>
</div>
`;
